sources/parsers/parsers.ml

```diff
@@ -47,37 +47,63 @@ let register_parser ~lang new_parser =
     end;
   parsers := (lang, new_parser) :: !parsers
 
+let debug_no_parser_for_no_extension () =
+  if verbose() then
+    eprintf
+      "error: no extension found, can't infer input format@."
```

Comment:
Errors, and even more importantly, fatal errors (after which alt-ergo exits) shouldn't print only in verbose mode I think.